### DISCUSSION TITLE
Bitwarden Desktop v2024.12.0 and newer.

### DISCUSSION THREAD

**Discussion by skarpinis** [dani-garcia/vaultwarden#5253] [General]
Added SSH-Key storage support. Currently only usable with Bitwarden Desktop v2024.12.0 and newer. Can I ask where i can get this version??? thanks

**Comment by BlackDex** [comment]
Wait for the official release from Bitwarden. Vaultwarden just already supports this feature.

**Reply by skarpinis** [reply]
ok ok :)
